proto/info/info.proto

```diff
@@ -635,6 +635,14 @@ message ZInfoDevice {
   bool remote_access_disabled = 52;
 }
 
+// OptionalCapabilities indicates any additional capabilities device wants
+// to publish to controller. For example Kubevirt hypervisor is not supported by
+// all eve flavors.
+message OptionalCapabilities {
+  // Virtualization type Kubevirt
+  bool HVTypeKubevirt = 1;
```

Comment:
Can you do this in lower_snake_case as yetus suggests?
protoc will make hv_type_kubevirt into HvTypeKubevirt in the golang definition.

Reply:
Done, amended the commit